lib/jekyll/drops/drop.rb

```diff
@@ -137,6 +137,7 @@ def inspect
         require 'json'
         JSON.pretty_generate to_h
       end
+      alias_method :to_json, :inspect
```

Comment:
* Use alias without symbolizations: `alias method other_method`
* Group `alias_method` at the bottom of the class rather than close to the method.
* Use alias instead of alias method.